frb_codegen/src/generator/dart/func.rs

```diff
@@ -169,3 +178,31 @@ pub(crate) fn generate_api_func(
         companion_field_implementation,
     }
 }
+
+pub(crate) fn generate_opaque_getters(ty: &IrType) -> GeneratedApiFunc {
+    let signature = format!(
+        "
+    DropFnType get drop_opaque_{0};
```

Comment:
uber-nit: Dart like `camelCase` not `snake_case`, so would be great if it is easy to change, but no worries if not easy